smartsim/settings/builders/launch/alps.py

```diff
@@ -29,15 +29,20 @@
 import typing as t
 
 from smartsim.log import get_logger
+from smartsim.settings.dispatch import ShellLauncher, default_dispatcher
 
 from ...common import StringArgument, set_check_input
 from ...launchCommand import LauncherType
 from ..launchArgBuilder import LaunchArgBuilder
 
+if t.TYPE_CHECKING:
+    from smartsim.settings.builders.launchArgBuilder import ExecutableLike
+
 logger = get_logger(__name__)
 
 
-class AprunArgBuilder(LaunchArgBuilder):
```

Comment:
Why have you added t.Sequence[str] to the class inheritance?

Reply:
Because I decided to make `LaunchArgBuilder` a generic class, and the type checker unfortunately cannot infer the types of the generics :(

Basically this just says that `AprunArgBuilder.finalize` will return a string, whereas `DragonArgBuilder.finalize` will return a `DragonRunRequestView`.

Reply:
The generic for `LaunchArgBuilder` has since been removed! Marking resolved!!